paywall/src/utils/keyExpirationTimestampFor.ts

```diff
@@ -1,3 +1,6 @@
+export const NO_SUCH_KEY =
+  '0x08c379a0000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000134841535f4e455645525f4f574e45445f4b455900000000000000000000000000'
```

Comment:
Where does this value come from?

Reply:
One of the smart contract versions returns `3963877391197344453575983046348115674221700746820753546331534351508065746944` which is `NO_SUCH_KEY`.

Reply:
It's MAX_INT on the smart contract